proto/api.proto

```diff
@@ -9,6 +9,14 @@ import "google/api/visibility.proto";
 import "protoc-gen-openapiv2/options/annotations.proto";
 
 option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
+  security: {
+    security_requirement: {
+      key: "Authorization";
```

Comment:
nit: probably "Auth Header" would be more verbally consistently with other bloxroute docs. would also help distinguish which field is the actual header name and which field is the label that shows up in Swagger